lib/components/Carleton360/Carleton360.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react' // Required for JSX syntax
+import { PageHeader } from '../PageHeader/PageHeader'
+
+export interface Carleton360Props {
+  children?: React.ReactNode
+  headerType?: 'h1' | 'h2'
+}
+
+export const Carleton360Wrapper = ({ children, headerType = 'h2' }: Carleton360Props) => {
+  const $componentMarginY = `my-6 md:my-12 first:mt-0`
+
+  return (
+    <div
+      className={`cu-carleton360 relative flex items-center justify-center mx-auto px-8 overflow-hidden md:px-16 rounded-xl not-prose ${$componentMarginY} pt-10 pb-12  z-[2] max-w-4xl w-full  flex-col gap-2 bg-cu-black-50 cu-zero-first-last`}
+    >
+      <PageHeader
+        header={''}
+        as={headerType}
+        size={headerType === 'h1' ? 'lg' : 'md'}
+        isWhite={false}
+        isCenter
+        noUnderline
+      >
+        <img
+          className="mb-4 h-12 justify-center mx-auto"
+          src={'https://cu-production.s3.amazonaws.com/students/img/carleton360.svg'}
+          alt={'test'}
+        />
```

Comment:
Well want to host the svg ourselves and not rely on it existing in someone elses asset group.

Also this would be a good chance to look at making this a background element and we can avoid using img tag. At least worth the time to investigate.